float64[] times
float64[] north_positions
float64[] east_positions
---
float64[] latitudes
float64[] longitudes
float64[] altitudes
float64[] lat_velocities
float64[] lon_velocities
float64[] alt_velocities

